index.js

```diff
@@ -1,15 +1,69 @@
+const chalk = require('chalk')
 const parse = require('ndjson').parse
 const through = require('through2').obj
 const prettyFactory = require('pino-pretty')
+const { prettifyTime } = require('pino-pretty/lib/utils')
```

Comment:
This could prove to be a bad idea. You're requiring a utility that is internal to an external module. There is absolutely no guarantee that the function will always behave as it does currently.

Reply:
Yes, that is true. But considering that both the modules are in the same org I thought this might be okay as it'll just be duplicated code otherwise. Maybe we could pin the version of `pino-pretty`? Or do the require and method call in a try/catch block so that atleast this doesn't break entirely

Reply:
> This could prove to be a bad idea. You're requiring a utility that is internal to an external module. There is absolutely no guarantee that the function will always behave as it does currently.

Should I change this then?